crucible-llvm-cli/test-data/ub/README.md

```diff
@@ -0,0 +1 @@
+Tests for various kinds of undefined behavior, see ../../../crucible-llvm/doc/undefined-behavior.md.
```

Comment:
Consider adding a hyperlink here for `../../../crucible-llvm/doc/undefined-behavior.md`.